### PR TITLE
State install does not update platform project

### PR DESCRIPTION
**Pull request by MDrakos**
~~Merging PR is blocked until we can update the error messaging around camel builds in progress.~~

https://www.pivotaltracker.com/story/show/178222594

### CODE REVIEW

File: internal/runners/packages/packages.go [hunk 1]
```diff
@@ -139,6 +132,7 @@ func executePackageOperation(pj *project.Project, cfg configurable, out output.O
 	} else {
 		out.Print(locale.Tr(fmt.Sprintf("%s_%s", ns.Type(), operation), name))
 	}
+	out.Print(locale.Tl("operation_success_local", "Your local project has been updated. Run [ACTIONABLE]state push[/RESET] to save changes to the platform."))
```
Comment (Naatan) [inline internal/runners/packages/packages.go:135]: Let's move this to en-US.yaml (getting a bit long).

Please use:

> Your local project has been updated. 
> Run `[ACTIONABLE]state push[/RESET]` to save changes to the platform.
> Run `[ACTIONABLE]state history[/RESET]` to review your local changes.

File: internal/runners/push/push.go [hunk 2]
```diff
@@ -175,7 +176,11 @@ func (r *Push) Run(params PushParams) error {
 		}
 	}
 
-	r.Outputer.Notice(locale.Tr("push_project_created", r.project.URL(), lang.String(), langVersion))
+	if remoteExists {
+		r.Outputer.Notice(locale.Tl("push_project_existing", "Project at [NOTICE]{{.V0}}[/RESET] has been updated with local changes", r.project.URL()))
```
Comment (Naatan) [inline internal/runners/push/push.go:180]:
```suggestion
		r.Outputer.Notice(locale.Tl("push_project_existing", "Project at [NOTICE]{{.V0}}[/RESET] has been updated with local changes", r.project.URL()))
```
```suggestion
		r.Outputer.Notice(locale.Tl("push_project_existing", "The project at [NOTICE]{{.V0}}[/RESET] has been updated with your changes.", r.project.URL()))
```

File: locale/en-us.yaml [hunk 3]
```diff
@@ -791,7 +791,7 @@ installer_err_engine_unknown:
 build_status_unknown:
   other: "Build status could not be retrieved, please view the project at: [NOTICE]{{.V0}}[/RESET] for more information"
 build_status_in_progress:
-  other: "The project is currently building. Please visit [NOTICE]{{.V0}}[/RESET] to see the progress."
+  other: "The local project checkout is currently building. Please visit [NOTICE]{{.V0}}[/RESET] to see the progress."
```
Comment (Naatan) [inline locale/en-us.yaml:794]:
```suggestion
  other: "Your changes are currently be built remotely on the ActiveState Platform. Please visit [NOTICE]{{.V0}}[/RESET] to see the progress."
```

File: pkg/platform/model/projects.go [hunk 4]
```diff
@@ -194,6 +194,11 @@ func ProjectURL(owner, name, commitID string) string {
 	return url
 }
 
+// CommitURL creates a valid platform commit URL for the given commit
+func CommitURL(commitID string) string {
```
Comment (Naatan) [inline pkg/platform/model/projects.go:198]: Feels like the wrong place, but I currently can't think of a better one..

File: pkg/platform/model/projects.go [hunk 5]
```diff
@@ -194,6 +194,11 @@ func ProjectURL(owner, name, commitID string) string {
...
+	return fmt.Sprintf("https://%s/commit/%s", constants.PlatformURL, commitID)
```
Comment (Naatan) [inline pkg/platform/model/projects.go:199]: Use `constants.DashboardCommitURL`

File: pkg/platform/runtime/setup/setup.go [hunk 6]
```diff
@@ -282,7 +282,11 @@ func aggregateErrors() (chan<- error, <-chan error) {
 
 func (s *Setup) installArtifacts(buildResult *model.BuildResult, artifacts artifact.ArtifactRecipeMap, alreadyInstalled store.StoredArtifactMap, setup Setuper) error {
 	if !buildResult.BuildReady && buildResult.BuildEngine == model.Camel {
-		return locale.NewInputError("build_status_in_progress", "", apimodel.ProjectURL(s.target.Owner(), s.target.Name(), s.target.CommitUUID().String()))
+		messageURL := apimodel.ProjectURL(s.target.Owner(), s.target.Name(), s.target.CommitUUID().String())
+		if s.target.Owner() == "" && s.target.Name() == "" {
+			messageURL = apimodel.CommitURL(s.target.CommitUUID().String())
```
Comment (Naatan) [inline pkg/platform/runtime/setup/setup.go:287]: I don't think this URL shows build progress. Let's bring this up with Akash.

**Review by Naatan — CHANGES_REQUESTED** [review]

File: pkg/platform/runtime/setup/setup.go (hunk 6 above)
Comment (akivanc88) [reply pkg/platform/runtime/setup/setup.go:287]: We don't need to fix this in this story but we need a follow-up to get the UX before release.
Placeholder PT: https://www.pivotaltracker.com/story/show/178590174

We can leave it as-is for now

File: internal/runners/push/push.go [hunk 7]
```diff
@@ -175,13 +166,20 @@ func (r *Push) Run(params PushParams) error {
 		}
 	}
 
-	r.Outputer.Notice(locale.Tr("push_project_created", r.project.URL(), lang.String(), langVersion))
+	if remoteExists {
+		r.Outputer.Notice(locale.Tl("push_project_existing", "The project at [NOTICE]{{.V0}}[/RESET] has been updated with your changes.", r.project.URL()))
+	} else {
+		r.Outputer.Notice(locale.Tr("push_project_created", r.project.URL(), lang.String(), langVersion))
+	}
```
Comment (mdrohmann) [inline internal/runners/push/push.go:173]: :+1:

**Review by mdrohmann — DISMISSED** [review]

**Pull request merged**